package.nls.json

```diff
@@ -243,7 +243,7 @@
     "DataScienceSurveyBanner.bannerLabelYes": "Yes, take survey now",
     "DataScienceSurveyBanner.bannerLabelNo": "No, thanks",
     "InteractiveShiftEnterBanner.bannerMessage": "Would you like to run code in the 'Interactive' window (an IPython console) for 'shift-enter'? Select 'No' to continue to run code in the Python Terminal. This can be changed later in settings.",
-    "DataScience.restartingKernelStatus": "Restarting Jupyter Kernel",
+    "DataScience.restartingKernelStatus": "Restarting Kernel {0}",
```

Comment:
Removed the word `Jupyter`, we don't use jupyter when `connecting to kernel`
Also added the `{0}`, so we can display the name of the kernel when we're restarting it.